pkg/oc/cli/admin/prune/imageprune/prune.go

```diff
@@ -656,15 +658,20 @@ func (p *pruner) addDeploymentsToGraph(dmnts *kapisext.DeploymentList) []error {
 // Edges are added to the graph from each deployment config to the images
 // specified by its pod spec's list of containers, as long as the image is
 // managed by OpenShift.
-func (p *pruner) addDeploymentConfigsToGraph(dcs *appsapi.DeploymentConfigList) []error {
+func (p *pruner) addDeploymentConfigsToGraph(dcs *appsv1.DeploymentConfigList) []error {
 	var errs []error
 
 	for i := range dcs.Items {
 		dc := &dcs.Items[i]
 		ref := getRef(dc)
 		glog.V(4).Infof("Examining %s", getKindName(ref))
-		dcNode := appsgraph.EnsureDeploymentConfigNode(p.g, dc)
-		errs = append(errs, p.addPodSpecToGraph(getRef(dc), &dc.Spec.Template.Spec, dcNode)...)
+		internalDc := &appsapi.DeploymentConfig{}
```

Comment:
@soltysh this is ugly, but I expect this will go away when we switch the graph to external types